configs/benchmark/pytorch/pytorchxla_torchbench_config.py

```diff
@@ -119,3 +107,132 @@ def get_torchbench_config(
       task_gcp_config=job_gcp_config,
       task_metric_config=job_metric_config,
   )
+
+
+# Below is the setup for torchbench GPU run.
+def set_up_torchbench_gpu(model_name: str = "") -> Tuple[str]:
+  """Common set up for TorchBench."""
+  # TODO(piz): There is issue with driver install through fabric.
+  # Currently we use pre-installed driver to avoid driver reinstall.
+  nvidia_install_clean = (
+      "sudo /usr/bin/nvidia-uninstall",
+      (
+          'sudo apt-get -y --purge remove "*cuda*" "*cublas*" "*cufft*"'
+          ' "*cufile*" "*curand*" "*cusolver*" "*cusparse*" "*gds-tools*"'
+          ' "*npp*" "*nvjpeg*" "nsight*" "*nvvm*"'
+      ),
+      'sudo apt-get -y --purge remove "*nvidia*" "libxnvctrl*"',
+      "sudo apt-get -y autoremove",
+      "sudo rm -rf /usr/local/cuda*",
+  )
+  nvidia_driver_install = (
+      ("lsof -n -w /dev/nvidia* | awk '{print $2}' | sort -u | xargs -I {}" " kill {}"),
+      (
+          "wget -q"
+          " https://developer.download.nvidia.com/compute/cuda/12.1.0/local_installers/cuda_12.1.0_530.30.02_linux.run"
+      ),
+      (
+          "sudo service lightdm stop ; sudo sh cuda_12.1.0_530.30.02_linux.run"
+          " --silent --driver --toolkit --no-drm --override"
+      ),
+      "cat /var/log/nvidia-installer.log",
+  )
+  docker_cmds = (
+      " apt-get update && apt-get install -y libgl1 &&"
+      " pip install numpy pandas &&"
+      " pip install --pre torchvision torchaudio -i"
+      " https://download.pytorch.org/whl/nightly/cu121 &&"
+      " cd /tmp/ && git clone https://github.com/pytorch/benchmark.git &&"
+      f" cd benchmark && python install.py models {model_name} &&"
+      " cd /tmp/ && git clone https://github.com/pytorch/xla.git"
+  )
+  return (
+      "sudo apt-get install -y nvidia-container-toolkit",
+      (
+          "sudo docker pull"
+          " us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_12.1"
+      ),
+      (
+          "sudo docker run --gpus all -it -d"
+          " us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_12.1"
+      ),
+      (
+          "sudo docker exec -i $(sudo docker ps | awk 'NR==2 { print $1 }')"
```

Comment:
Hmm, tried to the `... --entrypoint bash gcr.io/image/url -c 'docker_cmds'` but didn't work as expected. Below is the command I tried: 
```
sudo docker run --gpus all -i -d --network host us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_12.1 --entrypoint /bin/bash us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.8_cuda_12.1 -c 'pip install --user --pre torchvision torchaudio -i &&  https://download.pytorch.org/whl/nightly/cu121'"
```
Do you have any hint on how to specify the command using `--entrypoint`?

Reply:
But I think it is better to seperate the `docker run` and `docker exec` since we need to copy experiment data out from the docker with `docker exec` during postprocess stage.

Reply:
Updated to use docker name instead of `awk`.